### PR TITLE
Add pagination on all titles page

### PR DESCRIPTION
**Pull request by RemcoSimonides**

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/title/list/list.component.ts [hunk 1]
```diff
@@ -39,22 +46,43 @@ export class ListComponent implements OnInit {
...
+      tap(res => this.nbHits = res.nbHits),
       pluck('hits'),
+      tap(movies => this.hitsViewed = this.hitsViewed + movies.length),
       map(result => result.map(movie => movie.objectID)),
       switchMap(ids => ids.length ? this.movieService.valueChanges(ids) : of([])),
-      map(movies => movies.sort((a, b) => sortMovieBy(a, b, this.sortByControl.value))),
-    );
+      // map(movies => movies.sort((a, b) => sortMovieBy(a, b, this.sortByControl.value))), // TODO issue #3584
+      tap(movies => this.movieSearchResultsState.next(this.movieSearchResultsState.value.concat(movies))),
+      tap(_ => setTimeout(() => this.scrollService.go(), 0))
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/marketplace/title/list/list.component.ts:65]: Needs to be executed after movies has been added to the `movieSearchResults` behaviorSubject

**Comment by fritzschoff** [comment]
@RemcoSimonides what is the state of this PR?

**Comment by RemcoSimonides** [comment]
It's acceptable to be reviewed and merged. However, we are also planning on refactoring scrolling overall. But let's review&merge this anyway today :)

**Review by fritzschoff — APPROVED** [review]

**Comment by RemcoSimonides** [comment]
Wait with merge before I confirm with Vincent whether this needs to be added to catalog too ;)

**Pull request merged**